src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -15,30 +31,50 @@
 			lg:tw-justify-between
 			lg:tw-align-center
 			"
-	>
-		<div class="tw-flex-grow">
-			<h4 v-if="creditLeft === promoAmount" class="tw-mb-1 tw-px-3">
-				Let's Get Started
-			</h4>
-			<h4 v-else-if="creditLeft > 0" class="tw-mb-1 tw-px-3">
-				Keep Going!
-			</h4>
-			<h4 v-else-if="creditLeft === 0" class="tw-mb-1 tw-px-3">
-				You Did It!
-			</h4>
-			<h3 class="tw-mb-2 tw-px-3">
-				You have ${{ creditLeft }} in credit left
-				<span v-if="campaignPartnerName"> from {{ campaignPartnerName }} </span>
-			</h3>
-			<kv-grid class="tw-grid-cols-2">
-				<kv-progress-bar
-					:value="percentageLeft"
-					class="tw-mb-1.5 lg:tw-mb-1 tw-ml-3 tw-col-span-1"
-					:aria-label="`You have $${ creditLeft } in credit left`"
-				/>
-			</kv-grid>
+		>
+			<div class="tw-flex-grow">
+				<h4 v-if="creditLeft === promoAmount" class="tw-mb-1 tw-px-3">
+					Let's Get Started
+				</h4>
+				<h4 v-else-if="creditLeft > 0" class="tw-mb-1 tw-px-3">
+					Keep Going!
+				</h4>
+				<h4 v-else-if="creditLeft === 0" class="tw-mb-1 tw-px-3">
+					You Did It!
+				</h4>
+				<h3 class="tw-mb-2 tw-px-3">
+					You have ${{ creditLeft }} in credit left
+					<span v-if="campaignPartnerName"> from {{ campaignPartnerName }} </span>
+				</h3>
+				<kv-grid class="tw-grid-cols-2">
+					<kv-progress-bar
+						:value="percentageLeft"
+						class="tw-mb-1.5 lg:tw-mb-1 tw-ml-3 tw-col-span-1"
+						:aria-label="`You have $${ creditLeft } in credit left`"
+					/>
+				</kv-grid>
+			</div>
+			<div class="lg:tw-mr-3 lg:tw-ml-0 tw-ml-3 lg:tw-pt-3">
+				<kv-ui-button
+					v-show="basketLoans.length > 0"
+					@click="$emit('show-basket')"
+				>
+					Checkout now
+				</kv-ui-button>
+			</div>
 		</div>
-		<div class="lg:tw-mr-3 lg:tw-ml-0 tw-ml-3 lg:tw-pt-3">
+		<!-- Only checkout button appears when there is a matching campaign with no promo credit -->
```

Comment:
This else will trigger in other cases. Currently the if above shows if no matching campaign and promo credit. That means this else will show if there is no promo credit or it's a matching campaign, it doesn't check for both. Do we only want to show this button "when there is a matching campaign with no promo credit" or are we okay with the two cases I just mentioned?

Reply:
I think it's correct to show the basket button if there are loans in the basket and the other conditions are not met. This will ensure visibility of the Basket button during general usage of page without promo credit applied.

@tiarascoleman You can change `v-else` to `v-else-if="basketLoans.length > 0"` and remove the check on the nested button below which will prevent just the blank green box from showing up.

Reply:
@dyersituations Thanks for the catch! Just updated Line 19, which is the v-if statement. The main focus is that when there is not any promo credit (usually in the case of there being a matching campaign), the progress bar should be hidden and when there is promo credit, the meter should appear (I believe). In other words, the focus is more on if promoCredit is present or not and hopefully this new iteration of the v-if statement reflects this! :)